client/src/rpc_response.rs

```diff
@@ -147,6 +147,7 @@ pub struct RpcTransactionStatusMeta {
 #[serde(rename_all = "camelCase")]
 pub struct RpcTransactionStatus {
     pub slot: Slot,
+    pub confirmations: Option<usize>,
```

Comment:
What does the JSON look like when `confirmations` is `None`? Is it still present in the response as null?

Reply:
I can handle it either way: (1) return `confirmations: null`; or (2) don't include the `confirmations` field. Do you have a preference?

Reply:
Slightly prefer (1)

Reply:
Cool, that’s how it is written now, so ready for another look